python/ray/tune/impl/tuner_internal.py

```diff
@@ -145,6 +137,16 @@ def __init__(
         if not trainable:
             raise TuneError("You need to provide a trainable to tune.")
 
+        if self._entrypoint == AirEntrypoint.TUNER and not isinstance(
+            self._run_config, ray.tune.RunConfig
```

Comment:
I think there might be an edge case where Train+Tune is used (so Tuner entrypoint), but the RunConfig used is the one set in the trainer (logic from `_choose_run_config`).

Reply:
oh goodness

Reply:
choosing to ignore this, since the `Tuner(trainer)` usage emits a deprecation warning already.